### PR TITLE
Accept width and height from query parameters

### PR DESCRIPTION
**Pull request by rishi-raj-jain**
Working example: https://layer0-docs-og-image-default.layer0.link/api?title=Layer0&width=1280&height=720

### CODE REVIEW

File: pages/api/index.js [hunk 1]
```diff
@@ -19,8 +19,8 @@ export default async function handler(req, res) {
     })
   )
   await page.setViewport({
-    width: 1920,
-    height: 1080,
+    width: parseInt(width) || 1920,
```
Comment (jeffpatzer) [inline pages/api/index.js:22]: I'd move the var declaration into the destructure. Makes it easier to know there is a default.

File: pages/api/index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { generatePage } from '@/lib/page'
 import chromium from 'chrome-aws-lambda'
 
 export default async function handler(req, res) {
-  const { title } = req.query
+  const { title, width, height } = req.query
```
Comment (jeffpatzer) [inline pages/api/index.js:6]:
```js
const { title, width = 1920, height = 1080 } = req.query
```

File: pages/api/index.js (hunk 1 above)
Comment (rishi-raj-jain) [reply pages/api/index.js:22]: That's a great suggestion, thanks @jeffpatzer!

File: pages/api/index.js (hunk 2 above)
Comment (rishi-raj-jain) [reply pages/api/index.js:6]: Awesome, let me update this real quick.

File: pages/api/index.js (hunk 1 above)
Comment (rishi-raj-jain) [reply pages/api/index.js:22]: @jeffpatzer 
Apart from setting the default, the parameters needed to be parsed as numbers.
Resolved:
- https://layer0-docs-og-image-test.layer0.link/api?title=Solved,
- https://layer0-docs-og-image-test.layer0.link/api?title=Solved&width=1400&height=720

Comment (rishi-raj-jain) [reply pages/api/index.js:22]: Seems like I can't push to the branch anymore, @jeffpatzer can you update it to the following:

```js
// File: pages/api/index.js

var minify = require('html-minifier').minify
import { generatePage } from '@/lib/page'
import chromium from 'chrome-aws-lambda'

export default async function handler(req, res) {
  const { title, width = 1920, height = 1080 } = req.query
  const dom = generatePage(title)
  const browser = await chromium.puppeteer.launch({
    args: chromium.args,
    defaultViewport: chromium.defaultViewport,
    executablePath: await chromium.executablePath,
    headless: chromium.headless,
    ignoreHTTPSErrors: true,
  })
  const page = await browser.newPage()
  await page.setContent(
    minify(dom, {
      collapseWhitespace: true,
    })
  )
  await page.setViewport({ width: parseInt(width), height: parseInt(height) })
  const content = await page.$('body')
  const imageBuffer = await content.screenshot({ omitBackground: true })
  await page.close()
  await browser.close()
  res.setHeader(
    'Cache-Control',
    `public, immutable, no-transform, s-maxage=31536000, max-age=31536000`
  )
  res.setHeader('Content-Type', 'image/png')
  res.send(imageBuffer)
  res.status(200)
}
```

**Pull request closed without merging**